core/database/foxx/api/authz_router.js

```diff
@@ -74,22 +74,40 @@ router
             // Determine permissions associated with path provided
             // Actions: read, write, create, delete, chdir, lookup
             if (Object.keys(authzModule.authz_strategy).includes(req.queryParams.act)) {
-                authzModule.authz_strategy[req.queryParams.act][path_type](
-                    client,
-                    req.queryParams.file,
-                );
+                if (
+                    authzModule.authz_strategy[req.queryParams.act][path_type](
+                        client,
+                        req.queryParams.file,
+                        repo,
+                    )
+                ) {
+                    console.log(
+                        "AUTHZ act: " +
+                            req.queryParams.act +
+                            " client: " +
+                            client._id +
+                            " path " +
+                            req.queryParams.file +
+                            " SUCCESS",
+                    );
+                } else {
+                    console.log(
```

Comment:
Should this be console.error?
```suggestion
                    console.error(
```

Reply:
Good correction!